cloudify_aws/lambda_serverless/resources/function.py

```diff
@@ -85,11 +87,16 @@ def invoke(self, params):
                           % (self.type_name, params))
         res = self.client.invoke(**params)
         if res and res.get('Payload'):
-            res['Payload'] = res['Payload'].read()
+            try:
+                res['Payload'] = res['Payload'].read()
+                res['Payload'] = json.loads(res['Payload'].decode('utf-8'))
+                if res['Payload'].get('body'):
+                    res['Payload']['body'] = json.loads(res['Payload']['body'])
+            except json.JSONDecodeError:
+              pass
```

Comment:
@kfrukacz-gs You handled the response, but what about request ? does the json string passed correctly when you use paylaod from runtime properties ? and we need also to update the file-like object for payload

Reply:
@mabuaisha  working on it